intro_to_ada.rst

```diff
@@ -950,6 +950,93 @@ Since this value is not representable for the ``T3_D3`` type because the
 smallest value is 0.001, the actual value stored in variable ``A`` is
 zero.
 
+Fixed-point types
+-----------------
+
+Ordinary fixed-point types are similar to decimal fixed-point types.
+The difference between them is in the definition of the smallest
+representable value: for decimal fixed-point types, it is based on the
+power of ten, whereas for ordinary fixed-point types, it is based on the
+power of two. Therefore, they are also called binary fixed-point types.
+
+.. Probably remove this:
```

Comment:
On the contrary, that's context, and pretty useful IMHO to understand why two different kind of types exist. I would put that in a "going further" blockquote or something.